database/init/createTables.sql

```diff
@@ -49,10 +50,21 @@ create table volunteers (
     bio VARCHAR(150),
     active BOOLEAN,
     email VARCHAR(45) NOT NULL,
+    pronouns VARCHAR(15),
+    phone_number VARCHAR(15),
+    city VARCHAR(15),
+    province VARCHAR(15),
     FOREIGN KEY (fk_user_id) REFERENCES users(user_id)
     ON DELETE CASCADE
 );
 
+create table volunteer_profile_pics (
+    fk_volunteer_id VARCHAR(255) PRIMARY KEY,
+    profile_pic TEXT NOT NULL,
```

Comment:
i'm assuming you plan on storing the binary representation of images in the database, in this case we can use a [BLOB](https://learn.microsoft.com/en-us/sql/relational-databases/blob/binary-large-object-blob-data-sql-server?view=sql-server-ver16) type field. Unless this is meant to be a url field. Let me know what you had in mind.

Reply:
Yep sorry had it as text originally and forgot to change it. Just fixed it to blob